runway/hooks/staticsite/auth_at_edge/templates/http_headers/__init__.py

```diff
@@ -0,0 +1,23 @@
+
+# pylint: skip-file
+"""Add all configured headers (CloudFront compatable) to origin response.
+
+NOTE: pylint: skip-file has been added to this file to avoid an issue with a false
+positive about duplicated code. Please remove during iteration.
+"""
+from shared import as_cloud_front_headers, get_config  # pylint: disable=import-error
+
+
+def handler(event, _context):
+    """Handle adding the headers to the origin response.
+
+    Keyword Args:
+        event (Any): The Lambda Event
+    """
+    config = get_config()
```

Comment:
get_config() here should also be moved out of the handler.

Reply:
Moved